v2/apiserver/internal/core/jobs.go

```diff
@@ -262,6 +265,41 @@ func (j *jobsService) UpdateStatus(
 	return nil
 }
 
+func (j *jobsService) Cleanup(
+	ctx context.Context,
+	eventID string,
+	jobName string,
+) error {
+	event, err := j.eventsStore.Get(ctx, eventID)
+	if err != nil {
+		return errors.Wrapf(err, "error retrieving event %q from store", eventID)
+	}
+	_, ok := event.Worker.Jobs[jobName]
+	if !ok {
+		return &meta.ErrNotFound{
+			Type: "Job",
```

Comment:
Not sure if I've asked about his previously... do we have plans to create consts for supported types?  `const JobType string = "Job"`, etc.?

Reply:
We should probably do that.

Reply:
Created #1174 to track that.

Reply:
Thank you!